tests/dataprep/test_dataprep_milvus_langchain.sh

```diff
@@ -31,10 +31,7 @@ function build_docker_images() {
 
 function start_service() {
     # start milvus vector db
-    mkdir $WORKPATH/milvus
-    cd $WORKPATH/milvus
-    wget https://raw.githubusercontent.com/milvus-io/milvus/v2.4.9/configs/milvus.yaml
-    wget https://github.com/milvus-io/milvus/releases/download/v2.4.9/milvus-standalone-docker-compose.yml -O docker-compose.yml
+    cd $WORKPATH/comps/dataprep/milvus/langchain/
     sed '/- \${DOCKER_VOLUME_DIRECTORY:-\.}\/volumes\/milvus:\/var\/lib\/milvus/a \ \ \ \ \ \ - \${DOCKER_VOLUME_DIRECTORY:-\.}\/milvus.yaml:\/milvus\/configs\/milvus.yaml' -i docker-compose.yml
     docker compose up -d
```

Comment:
If not get files from milvus, then the docker compose file name is needed to modify as `docker-compose-dataprep-milvus.yaml`.
According to [comps/dataprep/milvus/langchain/docker-compose-dataprep-milvus.yaml](https://github.com/opea-project/GenAIComps/blob/main/comps/dataprep/milvus/langchain/docker-compose-dataprep-milvus.yaml)